ui/app/components/app/connected-accounts-list/connected-accounts-list-options-item/connected-accounts-list-options-item.component.js

```diff
@@ -0,0 +1,27 @@
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+
+export default class ConnectedAccountsListOptionsItem extends PureComponent {
+  static propTypes = {
+    iconClassNames: PropTypes.string,
```

Comment:
Should this be required? Or the existence of the `<i>` element be conditional on this prop being passed in? The `<i>` element doesn't really make sense to render without an icon class name.